tinymce/index.html

```diff
@@ -5,10 +5,11 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<title>TinyMCE Formatting Demo</title>
 		<link href="style.css" rel="stylesheet" type="text/css" media="all">
+		<link rel="stylesheet" type="text/css" href="../shared/index.css">
```

Comment:
Since we load the shared css, should we add the google font loading here to ensure the Noto font is loaded? 

`<link href="https://fonts.googleapis.com/css?family=Noto+Serif:400,400i,700,700i" rel="stylesheet">`